frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -135,6 +176,13 @@ async function verifyIdentity(userId: string, domain: string) {
     verifiedUserIdentity.value = (
         await idirBceidProxyApi.idirSearch(userId)
     ).data;
+
+    if(verifiedUserIdentity.value.found) {
+        userInfoStep.value.active = true;
+        userInfoStep.value.icon = IconSteps.checkmark;
+        addUserStep.value.active = addUserStep.value.icon === IconSteps.checkmark ? true : false;
```

Comment:
when verify the userId, it there a need to update the status of the 2rd or 3rd step?

Reply:
that was from the older code, I'll remove.